### ISSUE TITLE
How to get the accuracy of LFW to 99.80%?

### ISSUE THREAD

**Issue by WYao02** [nizhib/pytorch-insightface#9]
Hi, I test the pretrained iresnet50 on the LFW dataset and used standard pairs.txt from lfw, however, I only get an accuracy of 99.40%. I used the mtcnn+ retinaface to preprocess the LFW dataset(https://blog.csdn.net/qq_41684249/article/details/111302649).  Could you tell me how to do preprocess to get 99.80%? I tried many ways but still didn't get a similar result.

**Comment by nizhib** [comment]
Hi, unfortunately I wanted to create end2end evaluation pipeline for these models but never did.

All the scores reported are copied from the original insightface repo and I believe they should be the same for this one as I achieved `np.allclose` to be `True` for all the outputs after the weights are converted.

You may have luck researching the original mxnet-based `insightface` repo successfully for the evaluation code.
